charts/sourcegraph/values.yaml

```diff
@@ -797,6 +797,46 @@ postgresExporter:
       cpu: 10m
       memory: 50Mi
 
+syntacticCodeIntel:
+  # -- Environment variables for the `syntactic-code-intel-worker` container
+  env:
+    NUM_WORKERS:
+      value: "1"
```

Comment:
What does this env variable control? We don't want to limit the concurrency of `scip-syntax` itself to 1, right?

Reply:
Frankly I'm unsure. I've only set this to 1 for the local testing with Kind (hence the draft PR), as I expect this will wreck my laptop if it controls number of container copies

Reply:
I've removed it, couldn't find any references of it anywhere in the codebase that weren't referring back to this usage..